### ISSUE TITLE
Add Speed playback as an option

### ISSUE THREAD

**Issue by gatosyperros** [ajayyy/DeArrow#210]
Many YouTube videos can be played at 1.5-1.75 speed if not more depending on the YouTuber.
Many YouTuber do use good titles, it would be nice if we could add a suggested playback speed "prompt"  (since that's not the addon for that) to the tittle and or description.
some people just do not understand that this whole line of words could have been said way faster by just:
talking faster and or don't talk?

With sponsor block you can cut 1/4 of avideo and with speed at 1.5-2 you can cut another 30-15 minutes bringing down a 1h video to 30-35 minutes.

I don't want to speed run YouTube, but god damn I filler episodes is just not my thing.

TLDR: add speed playback as an option
